### PR TITLE
Added new compute quota metrics (quotas for the tenant)

### PR DESCRIPTION
**Pull request by pnordh**
New metrics:
```
kos_compute_quota_cores
kos_compute_quota_floating_ips
kos_compute_quota_instances
kos_compute_quota_ram_megabytes
```

The new metrics follows the same pattern as the already existing
`kos_cinder_quota_*` metrics, i.e they also have a `quota_type`
dimension.

Signed-off-by: Pontus Nordh <me@nordh.dev>

### CODE REVIEW

**Comment by pnordh** [comment]
@tobiasgiese @seanschneeweiss :)

File: pkg/metrics/nova.go [hunk 1]
```diff
@@ -76,6 +113,16 @@ func PublishServerMetrics(client *gophercloud.ServiceClient, tenantID string) er
 		publishServerMetric(srv)
 	}
 
+	// get compute quotas form openstack
```
Comment (tobiasgiese) [inline pkg/metrics/nova.go:116]:
```suggestion
	// Get compute quotas from OpenStack.
```

**Review by tobiasgiese — APPROVED** [review]
Just a typo. Rest LGTM :)
Thanks for your great contribution... and sorry for the delay 🙂

**Comment by tobiasgiese** [comment]
The build job failed because of an outdated go version. I'll create a PR to fix the job.

https://github.com/mercedes-benz/kosmoo/blob/35af45301375846bc72517d464c84d0de56b52c6/.github/workflows/ci.yml#L30

**Comment by tobiasgiese** [comment]
https://github.com/mercedes-benz/kosmoo/pull/95 was successfully merged. Please rebase against the `master` branch @pnordh :)

**Comment by pnordh** [comment]
> #95 was successfully merged. Please rebase against the `master` branch @pnordh :)

done :)

**Review by seanschneeweiss — APPROVED** [review]
Thanks a lot.

**Comment by pnordh** [comment]
@tobiasgiese thanks - can you push a new release tag as well?

**Comment by tobiasgiese** [comment]
> @tobiasgiese thanks - can you push a new release tag as well?

Sure, will do a new release the next days

**Pull request merged**